app/models/concerns/normalize_column.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module NormalizeColumn
+  extend ActiveSupport::Concern
+
+  included do
+    def self.normalize_column(column_name)
+      before_validation -> do
+        value = self[column_name]
+        if value
```

Comment:
```suggestion
        return unless value
```

Reply:
`return` won't work here as it's not a method, but inside a block. `next unless value` should do.

I'll check. Thanks!